src/cljc/harja/domain/tilannekuva.cljc

```diff
@@ -74,7 +74,10 @@
   [l-ja-p-alueiden-puhdistus "l- ja p-alueiden puhdistus"
    "L- ja P-alueiden puhdistus"]
   [muu "muu" "Muu"]
-  [liuossuolaus "liuossuolaus" "Liuossuolaus"]
+  ;; Liuossuolausta ei ymmärtääkseni enää seurata, mutta kesälomien takia tässä on korjauksen
```

Comment:
Joo, hyvä näin. Kunhan asia selvenee niin poistetaan tosiaan nämä kommenttirivitkin, kumpi tahansa on lopputulos.